### DISCUSSION TITLE
Implement for IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by DanielHGimenez** [daltonmenezes/aura-theme#127] [Port request]
Implement the theme for IntelliJ IDEA

**Comment by bearkfear** [comment]
I use android studio, i will be veryyyyyy happy to have this theme there. PLEASSEEEEE <3

**Comment by daltonmenezes** [comment]
Hi @bearkfear , open an issue only for Android Studio please, the ideal is one port per issue to make the contributions better and easier to be done. 💜

**Comment by PhyberApex** [comment]
I guess the help wanted label means you're open on a PR for this?

**Comment by daltonmenezes** [comment]
Hi @PhyberApex , that's right. It's about pull request. The Visual Studio Code port must be used as the guide to put the colors in a right way. If you would like to make it, just let me know to assignee you to this issue and let me know if you need some help. 💜

**Comment by trzpiot** [comment]
Hi @daltonmenezes, I can give it a try. 😃

**Comment by daltonmenezes** [comment]
@trzpiot sure, The Visual Studio Code port must be used as the guide to put the colors in a right way. Also, don't forget to read the [docs](https://github.com/daltonmenezes/aura-theme#documentation) and anticipate the knowledge about the [checklist](https://github.com/daltonmenezes/aura-theme/blob/main/.github/PULL_REQUEST_TEMPLATE.md) that PR must have. I'm going to assign you to this issue, just let me know if you need any help 💜

**Comment by jbrooksuk** [comment]
I've just used [JetBrains/colorSchemeTool](https://github.com/JetBrains/colorSchemeTool) to generate a converted version of aura-theme. You can get it here: https://gist.github.com/jbrooksuk/af4bd3a8292e713a0f45f60406acb3c1

**Comment by trzpiot** [comment]
Sorry that nothing has come so far.
@jbrooksuk, thanks! 🚀 Do you want to create the port as well or should I do it? Unfortunately I can't make any time promises, but I'll try my best.

**Comment by jbrooksuk** [comment]
@trzpiot it's not something I'm going to pursue, sorry. I'm not super familiar with the theming of IntelliJ editors. I just converted it quickly - it's definitely not perfect.

**Comment by trzpiot** [comment]
Hi @daltonmenezes,

unfortunately I won't manage to build the port in the near future, which is why I'm taking myself out as the assignee. Sorry folks. 😢

**Comment by DaBultz** [comment]
Hi, is anyone currently working on this? 

Would like to learn how to theme IntelliJ and love the theme in VSCode (Soft Text) and would love to also use it in IntelliJ Products.

**Reply by daltonmenezes** [reply]
@DaBultz You can work on this for sure! It would be awesome. 💜

**Reply by DaBultz** [reply]
Awesome!, will get started :)

**Comment by shoaib30** [comment]
Has anyone managed to port this to IntelliJ?

**Reply by jbrooksuk** [reply]
> Has anyone managed to port this to IntelliJ?

https://github.com/daltonmenezes/aura-theme/discussions/127#discussioncomment-2211556

**Comment by ArielFalcon** [comment]
> Implement the theme for IntelliJ IDEA

please

**Comment by Nicoss54** [comment]
Hey Any news about this @DaBultz ?

**Comment by oszoou** [comment]
I created a plugin for it.
https://plugins.jetbrains.com/plugin/27073-aura-theme

**Reply by DanielHGimenez** [reply]
I tried to download it, but I found some problems. I created the following issue in the repository with evidence of the problem. https://github.com/Oszoou/aura-theme-intellij/issues/1
